### DISCUSSION TITLE
Come si inserisce se un alunno viene bocciato ?

### DISCUSSION THREAD

**Discussion by gianlucaaiello56** [iisgiua/giuaschool#514] [Richieste di aiuto] [answered]
Come inserire "l'alunno non è stato ammesso alla classe successiva"
@trinko

**Comment by trinko** [comment]
Nello scrutinio finale, al passo ESITI, si inserisce l'esito NON AMMESSO.
Viene richiesta anche la motivazione da riportare a verbale.

**Comment by trinko** [comment, marked as answer]
Vedi anche il manuale presente nella documentazione, anche se non è aggiornato alle ultime modifiche:
[Scrutinio finale](https://iisgiua.github.io/giuaschool-docs/assets/docs/giuaschool_manuale_scrutinio_finale_coordinatori.pdf)

**Comment by gianlucaaiello56** [comment]
gentilissimo grazie
